### DISCUSSION TITLE
How to remove/alter the highlight around h* elements in a markdown file?

### DISCUSSION THREAD

**Discussion by sukantamaikap** [LazyVim/LazyVim#2497] [Q&A] [answered]
I'm using lazyvim with mostly out-of-the-box [setup](https://github.com/sukantamaikap/my.lazyvim). For the markdown files, lazyvim shows this highlighted area around the `h*` elements, which, in my opinion, looks loud. Here are some screenshots of what I'm talking about:

<img width="1291" alt="Screenshot 2024-02-04 at 6 49 25 pm" src="https://github.com/LazyVim/LazyVim/assets/3388408/8a197ff5-f97c-479a-9f16-623cd829cadd">

Theme: `catppuccin-frappe`

<img width="1288" alt="Screenshot 2024-02-04 at 6 49 49 pm" src="https://github.com/LazyVim/LazyVim/assets/3388408/51ab2be5-93b1-439b-bb31-dbc8b1219e0d">

Theme: `blue`

<img width="1373" alt="Screenshot 2024-02-04 at 6 54 41 pm" src="https://github.com/LazyVim/LazyVim/assets/3388408/204ebbb9-09f2-46a1-9927-5433486ee0cf">

Theme: `myrphy`

Also, please do note, there is no way to differentiate between `h*` elements and code blocks. How can I remove/reduce the highlight around the h* elements?

I'm not sure if it is relevant, but here is my LspInfo:

```vim
 Language client log: /Users/sm/.local/state/nvim/lsp.log
 Detected filetype:   markdown
 
 3 client(s) attached to this buffer: 
 
 Client: marksman (id: 1, bufnr: [28, 3])
 	filetypes:       markdown, markdown.mdx
 	autostart:       true
 	root directory:  /Users/sm/sandbox/web/sukantamaikap.com
 	cmd:             /Users/sm/.local/share/nvim/mason/bin/marksman server
 
 Client: typos_lsp (id: 2, bufnr: [28, 3])
 	filetypes:       *
 	autostart:       true
 	root directory:  Running in single file mode.
 	cmd:             /Users/sm/.local/share/nvim/mason/bin/typos-lsp
 
 Client: copilot (id: 3, bufnr: [28, 3])
 	filetypes:       
 	autostart:       false
 	root directory:  /Users/sm/sandbox/web/sukantamaikap.com
 	cmd:             node /Users/sm/.local/share/nvim/lazy/copilot.lua/copilot/index.js
 
 Configured servers list: lua_ls, marksman, ruff_lsp, tsserver, typos_lsp, yamlls, tailwindcss, jsonls, docker_compose_language_service, dockerls, terraformls, eslint, pyright
```

**Comment by dpetka2001** [comment, marked as answer]
`headlines.nvim` is the plugin you want to disable if I'm not mistaken.

**Reply by sukantamaikap** [reply]
@dpetka2001 you are a legend. Thanks a lot.
